go.mod

```diff
@@ -9,3 +9,7 @@ require (
 	github.com/paketo-buildpacks/packit v1.3.1
 	github.com/sclevine/spec v1.4.0
 )
+
+replace github.com/paketo-buildpacks/packit => /home/ubuntu/workspace/paketo-buildpacks/yarn/../packit
```

Comment:
Ignore: The purpose of this was to use a local version of packit which contained the SBoM changes